flask_resty/view.py

```diff
@@ -107,8 +108,62 @@ def validate_request_id(self, data, expected_id):
         if id != expected_id:
             raise ApiError(409, {'code': 'invalid_id.mismatch'})
 
+    def update_spec(self, path, spec, many=False, tag=False, **kwargs):
```

Comment:
a couple of questions with this.

- would you do the same for pagination, sorting, filtering?
- Does `ModelApiSpec` in your example return a function or the actual object? Because it needs a lot of parameters.

The idea of inheritance is that it's easy to call each superclass in order and each superclass has a chance of changing apispec, it's almost as update_spec is a more formal docstring. I'll experiment with what you are proposing, to see how the api turns out